packages/frontend/package.json

```diff
@@ -31,7 +31,8 @@
         "@hello-pangea/dnd": "^16.5.0",
         "@hookform/error-message": "^2.0.0",
         "@lightdash/common": "workspace:*",
-        "@lightdash/mantine-v7": "workspace:*",
+        "@mantine-8/core": "npm:@mantine/core@8.0.0",
+        "@mantine-8/hooks": "npm:@mantine/hooks@8.0.0",
```

Comment:
oh and another difference from the previous PR is that mantine is not in alpha anymore so it references 8.0.0 instead of 8.0.0-alpha.3